docs/pages/careers/design-engineer-x-grid.md

```diff
@@ -0,0 +1,109 @@
+# Design Engineer
+
+<p class="description">You will focus on design to implement great product experience for the MUI X Data Grid.</p>
```

Comment:
```suggestion
<p class="description">You will design and implement a great user and developer experience for the MUI X Data Grid.</p>
```

I think there's room to improve here — "product experience" sounds a bit too broad, and we can mention the user (as in the final) and the developer as targeted personas!